js/account_settings_disable_add_user_and_course.js

```diff
@@ -0,0 +1,42 @@
+function isAddUserButtonPresent(){
+    $btn = $('a.add_user_link');
```

Comment:
Maybe these could be more specific, targeting just `$('#right-side a.add_user_link')`? Selector would be quicker, and less chance they'll disable something else on another page if Canvas happens to reuse that class somewhere.